addons/portal/static/src/js/portal_chatter.js

```diff
@@ -136,6 +131,31 @@ var PortalChatter = publicWidget.Widget.extend({
     // Private
     //--------------------------------------------------------------------------
 
+    /**
+     * Reloads chatter and message count after posting message
+     *
+     * @private
+     */
+    _reloadChatterContent: function (data) {
+        this.messageFetch();
+        this._reloadComposer();
+    },
+    /**
+     * Destroy current composer widget and initialize and insert new widget
+     *
+     * @private
+     */
+    _reloadComposer: function () {
+        // destroy existing composer
+        if (this._composer) {
+            this._composer.destroy();
+        }
+        // instanciate and insert composer widget
```

Comment:
Considering how short and clear the code is, and that it's already documented in the docstring, these comments are redundant